test/parallel/test-runner-run.mjs

```diff
@@ -69,10 +69,12 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
   });
 
   it('should support timeout', async () => {
-    const stream = run({ timeout: 50, files: [
-      fixtures.path('test-runner', 'never_ending_sync.js'),
-      fixtures.path('test-runner', 'never_ending_async.js'),
-    ] });
+    const stream = run({
```

Comment:
This file seems to have a number of stylistic changes. Was that intentional?

Reply:
Nope, I will remove unnecessary changes on the next commit!